server.go

```diff
@@ -270,3 +275,53 @@ func (c *Server) callTestClose() {
 	c.serviceManager.servicesMutex.Unlock()
 	wg.Wait()
 }
+
+// SetValidPeers sets the set of peers with which this server can communicate.
+func (c *Server) SetValidPeers(roster *Roster) {
+	newPeers := make(map[network.ServerIdentityID]struct{})
+
+	for _, node := range roster.List {
+		newPeers[node.ID] = struct{}{}
```

Comment:
`ID` is deprecated

more `ID`s in later code - not commented

Reply:
TIL - https://stackoverflow.com/questions/37320287/maptstruct-and-maptbool-in-golang

So I should get rid of my `map[]bool`...